### DISCUSSION TITLE
Returning to home position on completion of job.

### DISCUSSION THREAD

**Discussion by Geoffnut** [winder/Universal-G-Code-Sender#2610] [Help]
Back again with another question, I would like to know how to accomplish of returning to the home position on the completion of a job.
I have been playing with GRBLplotter and it has a setting for adding a header and footer in which I have use to do this. Could someone point old blind freddy here into what to do. Also at the moment the tool stops where it is after finishing.
Much appreciated Geoff

**Comment by breiler** [comment]
There is no such setting in UGS at the moment. You will need your CAM to add a G28 command at the end. Just remember to set a home position with G28.1 command.
